bin/update_documentation

```diff
@@ -32,13 +32,13 @@ To reproduce, run `sbt bench` (this will take a while).
   </tr>
   <tr>
     <td>ScalacNamer211</td>
-    <td>$ColdScalacNamer211</td>
-    <td>$HotScalacNamer211</td>
+    <td>1179.715 ± 2.818 ms</td>
+    <td>62.111 ± 0.099 ms</td>
```

Comment:
How will this file be updated when we change our test files?

Reply:
Will be addressed as part of https://github.com/twitter/reasonable-scala/issues/3.